src/modules/account/components/overview-stats/overview-stats.styles.less

```diff
@@ -9,6 +9,19 @@
     margin-bottom: 0.375rem;
   }
 
+  @media @breakpoint-mobile {
+    > div {
+      > span:first-of-type {
+        font-size: @size-10;
```

Comment:
use the saved font-styles here instead of directly changing font-size, same for below (the .text/.mono stuff)